docker/linux/Dockerfile.centos

```diff
@@ -1,11 +1,11 @@
 # -------------------------------
 # Stage 1: Build cura-build-environment
 #
-FROM centos:centos7
+FROM centos:centos8
```

Comment:
The ticket states that we should move to Debian Stable, what is the reason we're staying on CentOS?